### PR TITLE
Change date time component page ID's for automation tests

### PR DESCRIPTION
**Pull request by MatthewAlexanderBeck**

### CODE REVIEW

**Comment by MatthewAlexanderBeck** [comment]
Image to show working validatio
Screenshot from 2022-08-18 15-59-27
n

File: app/views/components/InputDateTime.scala.html [hunk 1]
```diff
@@ -18,35 +18,95 @@
...
 
 @this(
-    heading: Heading,
-    inputDate: InputDate,
-    inputTime: InputTime
+        heading: Heading,
+        inputDate: InputDate,
+        inputTime: InputTime,
+        govukDateInput: GovukDateInput,
+        govukRadios: GovukRadios
```
Comment (ajchilts) [inline app/views/components/InputDateTime.scala.html:25]: Do we use this? Or can it go?

**Review by ajchilts — DISMISSED** [review]
Just one teeny comment @MatthewAlexanderBeck :-)

**Review by ajchilts — COMMENTED** [review]
LGTM 👍

**Review by ajchilts — APPROVED** [review]

**Pull request merged**